### PR TITLE
Update events.ts(ToA)

### PR DESCRIPTION
**Pull request by antisocial93**
First of all, i need to say that it's my first time pulling request on commiting changes on else's repo, so if anything - i'm sorry.

I added "Tower of Adversity: Hazard Zone" event's time.
I don't know really which ID should be, so just typed one almost random, and took the pic of in-game title of hazard zone and downloaded it to my repo, hope this is ok, 'cause didn't find a better one(tho tried to search in wuwa api).

### CODE REVIEW

File: events.ts [hunk 1]
```diff
@@ -101,6 +101,13 @@ const rawEvents: { banners: Event[]; activities: Event[] } = {
       startDate: '2024-06-13 04:00',
       endDate: '2024-06-27 03:59',
     },
+    {
+	  id: 10250,
```
Comment (Luzefiru) [inline events.ts:105]: You can leave the ID blank if you're unsure. Otherwise, LGTM.

**Review by Luzefiru — CHANGES_REQUESTED** [review]

**Comment by Luzefiru** [comment]
Please remove the `id` value before I can merge this. Thanks!

**Review by Luzefiru — CHANGES_REQUESTED** [review]
Please remove the entire `id` line. 😅

**Comment by antisocial93** [comment]
Mb, haha, done

**Comment by antisocial93** [comment]
Glad to help, even if it's so small

**Comment by antisocial93** [comment]
I just saw my pic quality on site and this was sooo awful, but after that remembered that nowadays there are plenty of photo enchancers, so it looks much better now

**Comment by Luzefiru** [comment]
> I just saw my pic quality on site and this was sooo awful, but after that remembered that nowadays there are plenty of photo enchancers, so it looks much better now

If you find a better alternative, feel free to make another PR. 👍

**Pull request merged**